src/main/java/com/spartronics4915/frc2020/subsystems/LED.java

```diff
@@ -0,0 +1,100 @@
+package com.spartronics4915.frc2020.subsystems;
+
+import com.spartronics4915.lib.util.Logger;
+import com.spartronics4915.lib.subsystems.SpartronicsSubsystem;
+
+import edu.wpi.first.wpilibj.SerialPort;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/**
+ * The subsystem that controls the LED.
+ * Please see the README.md in the Arduino_Bling directory for details.
+ */
+public class LED extends SpartronicsSubsystem
+{
+    private SerialPort mBling;
+
+    private static LED sInstance = null;
+    private static BlingState mBlingState;
+
+    public static LED getInstance()
```

Comment:
Sure - I can make the change, unless you want to take this on as I didn't notice where subsystems were getting created.

Reply:
Actually thinking about this, it is better to have LED subsystem as a singleton. This will ensure only one thread has access to the serial port for writes. Recommend leaving it as is as it could lead to unpredictable results.